en/docs/develop/scim2-patch-operations.md

```diff
@@ -0,0 +1,928 @@
+
+    ```json
+    {
+        "op": "replace",
+        "value": {
+            "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User": {
+                "country": "USA"
+            }
+        }
+    }
+    ```
+
+-   **Example 4:** Replace the `country` attribute in the **Enterprise** user schema (using the path param):
+
+    ```json
+    {
+        "op": "replace",
+        "path": "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:country",
+        "value": "UK"
+    }
+    ```
+
+**Complex singular attributes**
+
+Consider the `name` attribute in **Core** user schema, which has sub-attributes such as `givenName`, `familyName`, etc. and the `manager` attribute in the **Enterprise** user schema (`urn:ietf:params:scim:schemas:extension:enterprise:2.0:User`), which has `displayName`, `value`, `ref`, etc. as sub-attributes. 
+
+-   **Example 1:** Replace the `name` attribute in the **Core** user schema (without using the path param):
+
+    ```json
+    {
+        "op": "replace",
+        "value": {
+            "name": {
+                "givenName": "Peterson"
+            }
+        }
+    }
+    ```
+
+-   **Example 2:** Replace the `name` attribute in the **Core** user schema (using the path param):
+
+    ```json   
+    {
+        "op": "replace",
+        "path": "name",
+        "value": {
+            "givenName": "Martin",
+            "familyName": "Freeman"
+        }
+    },
+    {
+        "op": "replace",
+        "path": "name.familyName",
+        "value": "Jackson"
+    }
+    ```
+
+-   **Example 3:** Replace the `manager` attribute in the **Enterprise** user schema (without using the path param):
+
+    ```json
+    {
+        "op": "replace",
+        "value": {
+            "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User": {
+                "manager": {
+                    "displayName": "Manager3",
+                    "value": "Tom"
+                }
+            }
+        }
+    }
+    ```
+
+-   **Example 4:** Replace the `manager` attribute in the **Enterprise** user schema (using the path param):
+
+    ```json
+    {
+        "op": "replace",
+        "path": "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:manager",
+        "value": {
+            "displayName": "Manager4"
+        }
+    },
+    {
+        "op": "replace",
+        "path": "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:manager.value",
+        "value": "Jem"
+    }
+    ```
+
+**Simple multi-valued attributes**
+
+This type of attribute is not found in the core schemas of the SCIM specification. However, there is a capability to add simple multi-valued attributes to our extended schemas (custom schemas). 
+
+Let's consider a custom schema with an attribute called devices.
+
+-   **Example 1:** Replace the `devices` attribue in a **custom** user schema (without using the path param):
```

Comment:
```suggestion
-   **Example 1:** Replace the `devices` attribue in a **Custom** user schema (without using the path param):
```
Let's capitalize the `C` to match with Core and Enterprise keywords. Check other places as well